genjs.ml

```diff
@@ -1129,7 +1136,8 @@ let generate_enum ctx e =
 	| Some e ->
 		print ctx "%s.__meta__ = " p;
 		gen_expr ctx e;
-		newline ctx
+		newline ctx;
+	flush ctx
```

Comment:
Note that this is part of the `Some e` branch which the indent suggests it shouldn't be.